code/modules/mob/living/status_procs.dm

```diff
@@ -433,10 +433,10 @@
 /mob/living/proc/become_husk(source)
 	if(!HAS_TRAIT(src, TRAIT_HUSK))
```

Comment:
Putting `ADD_TRAIT(src, TRAIT_HUSK, source)` before this if statement, then putting update_body() back to where it was should fix this bug, now that I think about it. Will be a less demanding solution since update_body() is still only run the first time a body becomes a husk.

Reply:
yea do this